app.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once __DIR__ . '/autoload.php';
+
+use Codeception\Application;
+
+$app = new Application('Codeception', Codeception\Codecept::VERSION);
```

Comment:
I always prefer to do stuff like this in a closure to prevent having an `$app` in the global namespace:

```php
(function() {
$app = new App(...);
$app->modify();

return $app;

})()->run();

```

Reply:
Thanks, I will make this change later.

Reply:
@SamMousa that syntax does not work on PHP 5.6 https://3v4l.org/K2D73

Reply:
You can use `call_user_func` in php 5.6 I think

Reply:
Done

Reply:
What is the static keyword for in this context?

Reply:
PhpStorm suggested it as microoptimisation.

Reply:
> As of PHP 5.4, anonymous functions may be declared statically. This prevents them from having the current class automatically bound to them. Objects may also not be bound to them at runtime

Reply:
Thnx!